src/config.rs

```diff
@@ -1,14 +1,18 @@
+use std::path::PathBuf;
+
 #[derive(Debug)]
 pub struct Config {
     pub tmp_folder: String,
 }
 
-const DEFAULT_TMP_REPOSITORY: &str = "./tmp";
+const DEFAULT_TMP_REPOSITORY: &str = "tmp";
 
 impl Config {
     pub fn new() -> Config {
+        let dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
```

Comment:
Do you know what would happen if the user only had an executable and not the rust project ?